src/dune/build_context.ml

```diff
@@ -0,0 +1,13 @@
+open! Stdune
+open! Import
+
+type t =
+  { name : Context_name.t
+  ; build_dir : Path.Build.t
+  ; env : Env.t
+  ; ocaml : Action.Prog.t
```

Comment:
Why does ocaml need to be passed? I don't notice it being used.

Reply:
I could have sworn that something needed it, but on a rebuild it seems not. I've made the change.

Reply:
It seemed a bit suspicious because the build rules shouldn't know anything about OCaml rules.

Reply:
Agreed; I think it was leftover mistake from a previous pass I made that I overlooked.